src/ImageCropBundle/Bridge/Chameleon/BackendModule/ImageCropEditorModule.php

```diff
@@ -590,6 +591,23 @@ protected function saveCrop()
         );
     }
 
+    protected function deleteCrop() {
+        try {
+            $cropId = $this->inputFilterUtil->getFilteredInput(self::URL_PARAM_CROP_ID);
+            if (null !== $cropId) {
+                $this->imageCropDataAccess->deleteCrop($cropId);
+            }
```

Comment:
The method structure could be improved for readability:

- Reading $cropId doesn't need to be in the `try` block and should be moved to the begin of the method + early return in case of null (we could think about an error message in case of null so that we do not fail silently).
- `$this->returnAsAjaxResponse([])` can be added to the `try`block so we can easily spot that's the "return value" on success".

In the `catch` block, please declare the array explicitly (`$return = [ 'errorMessage' => ...]`).

Reply:
OK